files/es/learn/accessibility/mobile/index.md

```diff
@@ -217,7 +217,7 @@ Hemos proporcionado un ejemplo sencillo que muestra cómo usar el mouse y los ev
 
 En particular, los problemas más comunes que deben abordarse para dispositivos móviles son:
 
-- Adecuación de diseños para dispositivos móviles. Un diseño de varias columnas no funcionará tan bien en una pantalla estrecha, por ejemplo, y es posible que deba aumentar el tamaño del texto para que sea legible. Dichos problemas se pueden resolver creando un diseño sensible utilizando tecnologías tales como [mconsultas de medios, ](/es/docs/Web/CSS/Media_Queries)[vista](/es/docs/Mozilla/Mobile/Viewport_meta_tag) and [flexbox](/es/docs/Learn/CSS/CSS_layout/Flexbox). .
+- Adecuación de diseños para dispositivos móviles. Un diseño de varias columnas no funcionará tan bien en una pantalla estrecha, por ejemplo, y es posible que deba aumentar el tamaño del texto para que sea legible. Dichos problemas se pueden resolver creando un diseño sensible utilizando tecnologías tales como [mconsultas de mediosx](/es/docs/Web/CSS/Media_Queries), [vista](/es/docs/Mozilla/Mobile/Viewport_meta_tag) and [flexbox](/es/docs/Learn/CSS/CSS_layout/Flexbox). .
```

Comment:
typos from origin

```suggestion
- Adecuación de diseños para dispositivos móviles. Un diseño de varias columnas no funcionará tan bien en una pantalla estrecha, por ejemplo, y es posible que deba aumentar el tamaño del texto para que sea legible. Dichos problemas se pueden resolver creando un diseño sensible utilizando tecnologías tales como [consultas de medios](/es/docs/Web/CSS/Media_Queries), [vista](/es/docs/Mozilla/Mobile/Viewport_meta_tag) y [flexbox](/es/docs/Learn/CSS/CSS_layout/Flexbox). .
```

Reply:
I saw a few "and"s and thought about touching them 😄